tools/interop_matrix/create_matrix_images.py

```diff
@@ -260,11 +259,10 @@ def cleanup():
 
 def maybe_apply_patches_on_git_tag(stack_base, lang, release):
     files_to_patch = []
-    for release_info in client_matrix.LANG_RELEASE_MATRIX[lang]:
-        if client_matrix.get_release_tag_name(release_info) == release:
-            if release_info[release] is not None:
-                files_to_patch = release_info[release].get('patch')
-                break
+
+    release_info = client_matrix.LANG_RELEASE_MATRIX[lang][release]
```

Comment:
nit: when is `release_info` falsey?

Reply:
It can happen if release is "master" I think (in which case `[release]` is wrong and it should be `.get(release)`. I have a fix for that in a followup PR https://github.com/grpc/grpc/pull/17671/files#diff-6b34b224bd4d57f6dfdf3f46a282af44R39

Reply:
SG. Optional nit: might be worth adding an `if not release_info ... assert release == 'master'`